plugins/LilyPad/resource.h

```diff
@@ -91,18 +96,17 @@
 #define ID_RSTICK_DOWN                  0x1116
 #define ID_RSTICK_LEFT                  0x1117
 #define ID_ANALOG                       0x1118
-#define ID_DELETE                       0x11FF
-#define ID_DEBUG                        0x1200
-#define ID_IGNORE                       0x1201
-#define ID_CLEAR                        0x1202
-#define ID_REFRESH                      0x1202
+#define ID_IGNORE                       0x1119
+#define IDC_FORCEFEEDBACK               0x111A
+#define IDC_FORCEFEEDBACK_FUNCTION      0x111B
+#define ID_BIG_MOTOR                    0x111C
+#define ID_SMALL_MOTOR                  0x111D
+#define ID_FORCEFEEDBACK_BOX            0x111E
+#define IDC_CONFIGURE_ON_BIND           0x111F
+#define IDC_DEVICE_SELECT               0x1120
 #define ID_SAVE                         0x1204
 #define ID_LOAD                         0x1205
-#define IDC_LIST                        0x1207
-#define IDC_FORCEFEEDBACK               0x1208
-#define IDC_FORCEFEEDBACK_FUNCTION      0x1209
-#define ID_BIG_MOTOR                    0x120A
-#define ID_SMALL_MOTOR                  0x120B
+#define IDC_BINDINGS_LIST               0x1207
```

Comment:
Might be nice to follow arithmetic progression in the macro order but honestly I don't care.